packages/react-server/core/context/Navigator.js

```diff
@@ -50,7 +50,7 @@ class Navigator extends EventEmitter {
 
 		this._haveInitialized = true;
 
-		var route = this.router.getRoute(request.getUrl(), {navigate: {path:request.getUrl(), type:type}});
+		var route = this.router.getRoute(request.getUrl(), {method: request.getMethod()});
```

Comment:
I don't understand why it was necessary either, since it didn't really seem to be used. It _might_ have been used with `fluxible`, which `routr` was originally built for. But we basically picked the History API and `routr` from that project (and then modified the history API, IIRC) and called it a day. So it's not particularly surprising that we weren't using the extras on `route`.